tests/io-data.toit

```diff
@@ -0,0 +1,21 @@
+// Copyright (C) 2023 Toitware ApS.
+// Use of this source code is governed by a Zero-Clause BSD license that can
+// be found in the tests/LICENSE file.
+
+import io
+
+class FakeData implements io.Data:
+  data_ / ByteArray
+
+  constructor .data_:
+
+  constructor.str str/string:
+    data_ = str.to-byte-array
+
+  size -> int: return data_.size
```

Comment:
I'd probably put this on a new line for consistency.

Reply:
done.